### DISCUSSION TITLE
Issues with Danish keyboard and suggestions

### DISCUSSION THREAD

**Discussion by michaelbrandi** [community/community#15121] [Copilot Conversations]
The combination for looking at other suggestions, is supposed to be ALT+ [ however on a Danish keyboard, you are supposed to press ALT Gr to get the [ key, and Alt + Alt Gr + the key for [ (the same as the number 8) does not work, so I don't know how to cycle between code suggestions?

In Visual Studio 2022, it's supposed to be Ctrl + Alt + [ but the problem is the same, no combination works.

**Comment by k3b** [comment]
see https://github.com/orgs/github-community/discussions/6993
